dev/sbt-plugin/src/sbt-test/sbt-plugin/cassandra-overrides-default-configs/project/plugins.sbt

```diff
@@ -1 +1,2 @@
+addSbtPlugin("com.lightbend.lagom" % "lagom-sbt-plugin" % sys.props("project.version"))
```

Comment:
Adding `lagom-sbt-plugin` to make scripted tests more consistent with each other and also closer to how end-users projects.